vm/systemSmartContracts/validator.go

```diff
@@ -1695,6 +1698,18 @@ func (v *validatorSC) unBondTokens(args *vmcommon.ContractCallInput) vmcommon.Re
 func (v *validatorSC) unBondTokensFromRegistrationData(
 	registrationData *ValidatorDataV2,
 	valueToUnBond *big.Int,
+) (*big.Int, vmcommon.ReturnCode) {
+	isV1Active := !v.flagUnbondTokensV2.IsSet()
```

Comment:
why using double negation?
`if v2 -> return for v2` instead of `if !v2 -> return for v1`

Reply:
because this pattern will be extended to be used on v2, v3 and so on